### ISSUE TITLE
Decimal field gets overwritten by big decimal number

### ISSUE THREAD

**Issue by Yodale1000** [django-import-export/django-import-export#1425]
![grafik](https://user-images.githubusercontent.com/41866123/164006862-953071ea-8999-48fe-972b-81429d23669c.png)

**Comment by Yodale1000** [comment]
The number in xlsx is 1803,60

**Comment by matthewhegarty** [comment]
- What type is the underlying model field?
- Have you declared this field in your resource?  If so, what widget type have you used (if any)?
- The value is definitely `1803,60` ? (I get `decimal.ConversionSyntax` when trying to import this value)

**Comment by matthewhegarty** [comment]
I tried importing an XLSX field (Book "price" ) with a value of 1803.60 (I set the format to '0.00').  On, import the screen implies that the trailing '0' will be lost on import which seems incorrect, because the value has not changed:

![import1](https://user-images.githubusercontent.com/6249838/164038898-51151f85-b28b-4cdc-98d8-8c03dca7ccfc.png)

**Comment by Yodale1000** [comment]
mode field: `raw_requirement = models.DecimalField(
        verbose_name="Rohbedarf", max_digits=8, decimal_places=2, blank=True, null=True
    )`
resource field: `raw_requirement = fields.Field(column_name="Rohbedarf", attribute="raw_requirement")`
actual value in xlsx file: 1803,60 (cell formated as number)
version of django / django-import-export: 4.0.2 / 2.8.0

**Comment by matthewhegarty** [comment]
Can you confirm that when it is imported to the db the value is correct? (you have `decimal_places` set to 2)?

**Comment by Yodale1000** [comment]
Yes the value from the xlsx file gets saved correctly to the db. Apparently it is only wrong while importing.
